src/ontology/cl-edit.owl

```diff
@@ -4031,9 +4031,11 @@ SubClassOf(obo:CL_0000117 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000031))
 
 # Class: obo:CL_0000118 (basket cell)
 
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_0000118 "https://orcid.org/0000-0002-2825-0621")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Basket_cell&oldid=951703880") obo:IAO_0000115 obo:CL_0000118 "Basket cells are inhibitory GABAergic interneurons of the brain. In general, dendrites of basket cells are free branching and contain smooth spines. Axons are highly branched. The branched axonal arborizations give rise to basket-like structures that surround the soma of the target cell. Basket cells form axo-somatic synapses, meaning their synapses target somas of other cells.")
```

Comment:
not sure if the & is a problem here? it's highlighted in red

Reply:
Hi @shawntanzk and @dosumis ,
Shawn is right in that the WikipediaVersioned link format might cause issues. I've been using that pattern, as David suggested it was the correct format, but I see now that it doesn't resolve, see e.g. http://purl.obolibrary.org/obo/UBERON_8410017 (it doesn't seem to resolve in Protege either). I Googled the issue but didn't find an answer. It'd be great if we could come to a consensus on the format please, as I'm using it often. Thanks to all! :-)

Reply:
I think it requires this PR to be merged in order for resolution to work on OLS https://github.com/geneontology/go-site/pull/1607
(I don't know how this is configured in Protege - may be hard wired).

Reply:
Have merged.  Let's see if it fixes resolution the next time OLS is rebuilt.

Reply:
Not sure why the `&` shows up as red in the diff.

Reply:
@dosumis thanks. I'll make a ticket to remind myself to double-check that the WikipediaVersioned links resolve for my Uberon terms.I'll leave it to you or @shawntanzk to check the CL term(s) ;-)